arrayfire/library.py

```diff
@@ -57,7 +60,9 @@ class af_cdouble_t(ct.Structure):
         c_dim_t = c_int_t
 
 try:
+    # FIXME
```

Comment:
Whenever such FIXMEs are added can you please add what to fix there so that any other who wants pick up the task can understand what the label is about.

Reply:
You are right. Some variables are considered to be `None` by default, so it should be checked. Nice catch! I'll fix it.
In other cases, some of your checks, e.g., `AF_PATH` in `library.py` check if `AF_PATH is None`, but it should fail if `AF_PATH == ""`. Simple conditions change to `if AF_PATH` can fix such problems when the default value or variable type is not known.

Reply:
Fixed